packages/x-tree-view-pro/src/internals/plugins/useTreeViewItemsReordering/useTreeViewItemsReordering.itemPlugin.ts

```diff
@@ -34,6 +36,9 @@ export const useTreeViewItemsReorderingItemPlugin: TreeViewItemPlugin = ({ props
     itemId,
   );
   const isValidTarget = useSelector(store, selectorItemsReorderingIsValidTarget, itemId);
+  const draggedItemId = useSelector(store, selectorDraggedItem);
```

Comment:
This will cause every item to re-render everytime `draggedItemId` changes.

You could create a `selectorIsDraggedItemBeingEdited`, but same you will still re-render every item everytime you start dragging your edited item.

Could you instead prevent putting in the state an item that is being edited in the first place?

Reply:
Doesn't seem to solve the problem. The best thing I could think of is to disable dragging when editing is enabled on any item. WDYT?